mkdocs.yml

```diff
@@ -54,6 +54,7 @@ nav:
     - Deployments: 'Deployments.md'
     - Changelog: 'Changelog.md'
     - Visual Identity: 'Visual-Identity.md'
+    - Dictionary: 'Dictionary.md'
```

Comment:
Is this really user-facing documentation? To me it's more for developer and therefore should be in `doc/dev` like, for example, the naming conventions.